Milestone1

miftah1991 opened the pull request:
Setup And Mobile Version Skeleton (Protofoli Project) -Milestone 1 
I received the comments from my code reviewer and mistakenly closed the first pull request and now I realized that I was not supposed to close the pull request, in this pull request I made the below changes.

- Changed the color of the menu icon to white as per Figma standard
- managed the space  between header and content section
- removed extra space in my all files

miftah1991 committed:
Portfolio Project

This is my First Portfolio project

miftah1991 committed:
Adding node_module to .gitignore file

Adding node_module to .gitignore file

miftah1991 committed:
Configure Linters

Configure Linters

miftah1991 committed:
MileStone 1 Project Work

miftah1991 committed:
Solving Linters error

miftah1991 committed:
css error

miftah1991 committed:
CSS

CSS

miftah1991 committed:
Solving Linter Issues

Solving Linter Issues

miftah1991 committed:
Create screenshot of the work

miftah1991 committed:
Update README.md

Lordkaito commented on index.html:
```diff
@@ -0,0 +1,33 @@
+        <a href="#" class="mylogo">Miftah</a>
+        <nav class="mob-version">
+           <i class="fa fa-bars">
+
```
- [x] Please, try to avoid empty lines or blank spaces, as this is part of the [HTML && CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). Please make sure to check it on the entire project.

miftah1991 replied:
@Lordkaito Thank you so much for reviewing my code, Yes I noticed this mistake, I have reviewed the entire project and removed extra space.

Lordkaito commented on style.css:
```diff
@@ -0,0 +1,72 @@
+body {
+  background-color: #191765;
+  font-family: -apple-system, Raleway, Roboto, Oxygen, sans-serif;
+}
```
- [x] Please make sure your design matches the given Figma design as much as possible. This is regarding your body element. It has a surrounding white space that should not be placed there. So please make sure to check all the `padding` and `margin` rules and modify them if necessary to match the design as much as possible.

miftah1991 replied:
@Lordkaito, Appreciate your efforts, I just changed the margin and padding, and now it's the same as Figma design.

Lordkaito commented on index.html:
```diff
@@ -0,0 +1,33 @@
+    <header>
+        <a href="#" class="mylogo">Miftah</a>
+        <nav class="mob-version">
+           <i class="fa fa-bars">
```
- [x] Please make sure to use the same colors (or the most similar ones) as well as the fonts within your design to try to match the Figma template. Here, your hamburger menu is black while in the original design it's white. Kindly change it.

miftah1991 replied:
Changed the color, Sorry that I was not attentive to it.

Lordkaito reviewed (changes requested):
Hi @miftah1991, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

miftah1991 committed:
Changes of Review

removed extra space, changed the color of menu icon and managed space between header and content section

miftah1991 committed:
Merge branch 'Milestone1' of https://github.com/miftah1991/SetupAndMobileVersionSkeleton into Milestone1

miftah1991 commented:
Setup And Mobile Version Skeleton (Protofoli Project) -Milestone 1 
I received the comments from my code reviewer and mistakenly closed the first pull request and now I realized that I was not supposed to close the pull request, in this pull request I made the below changes.

- Changed the color of the menu icon to white as per Figma standard
- managed the space  between header and content section
- removed extra space in my all files
